syntax = "proto3";

package proto_msg;


option java_package = "com.github.cristobalm.RDFEWK2C.proto";
option java_outer_classname = "MsgType";

enum MessageType {
  UNKNOWN = 0;
  CONNECTION_END = 1;
  RUN_QUERY = 2;
  INVALID_QUERY_RESPONSE = 3;
  RESULT_TABLE_PART_RESPONSE = 4;
  RECEIVE_REMAINING_RESULT = 5;
  TIMED_OUT_RESPONSE = 6;
  CACHE_MISS = 7;
}
